src/routes/docs/+layout.svelte

```diff
@@ -12,6 +12,12 @@
   </div>
   <div class="w-full border-gray-300 dark:border-gray-600 lg:w-3/5 lg:border-x lg:px-8">
     <slot />
+    <div class="flex" style=" justify-content: space-around; margin-top:5%;">
+      <a
+        class="default-transition text-skin-text-highlight hover:underline hover:decoration-inherit"
```

Comment:
I think we should add the link to `/app` to the topbar, what do you think?

Reply:
yeah, then i think i should name it just "AppPage" and place it in topbar, or any other name you suggest.
<img width="924" alt="image" src="https://user-images.githubusercontent.com/95224348/223196316-7c1f06c2-8716-4e1d-b494-2b43ec53fdf8.png">